cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "https://candy.freefeed.net",
+  "baseUrl": "http://localhost:3333/",
```

Comment:
small concern here. probably case with running against local setup of FrF should be described in readme in this repo (or added some links at least).

Reply:
will make readme clearer on this in a separate PR

Reply:
Done https://github.com/FreeFeed/acceptance-tests/commit/a504f80103dd2c795030c191f64865c6bcec2afb